client/my-sites/marketplace/controller.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal Dependencies
```

Comment:
Let's keep the capitalization style consistent. By spot-checking a few files like https://github.com/Automattic/wp-calypso/blob/trunk/client/state/index.js#L2, I think this should be `Internal dependencies`